src/app/body/release/add-release/add-release.component.ts

```diff
@@ -147,35 +149,53 @@ export class AddReleaseComponent implements OnInit {
   }
 
   addRelease() {
-    this.showLoader = true;
+    this.showLoader = false;
     const projectLink=this.teamService.teamsDataJson[this.teamId].ProjectLink;
     this.gitToken = this.teamService.teamsDataJson[this.teamId].GitToken;
     this.gitToken = atob(this.teamService.teamsDataJson[this.teamId].GitToken);
     this.githubService.createGithubRelease(this.gitToken, this.releaseName, this.tagName, this.targetBranch, this.description, this.response2, this.response3, this.response1, projectLink).subscribe({
       next: (data) => {
-        this.addReleaseDetailsToDB();
-        this.showLoader = false;
+        // this.addReleaseDetailsToDB();
+        if(data['status']==201){
         this.popupHandlerService.addReleaseActive = false;
+        this.showLoader = false;
+        }
       },
-      error: (error) => {
-        console.error(error);
+      error: (data) => {
+        if(data['status']==403){
+          console.log("Token is expired");
+          this.tokenExpired=true;
+          this.showLoader = false;
+          let errorType = this.componentName + "_VALIDATION_TITLE";
+          this.errorHandlerService.addError(errorType, "Token expired")
+          return (false);
```

Comment:
errorCodes.xml file does not contain `ADDRELEASES` component 
please add that to get proper error code

Reply:
Better show it on the page directly that there is problem with token, rather than error code